<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SeanWang" name="Deals Test" tests="26" failures="18" timestamp="7 Apr 2018 15:07:31 GMT" time="32.252" errors="0">
  <testcase name="testGetDealByID" time="1.688" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testGetDealByID(DealsTestCases.java:54)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetDealByID -->
  <testcase name="testBulkDelete" time="1.985" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testBulkDelete(DealsTestCases.java:99)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testBulkDelete -->
  <testcase name="testGetDealByID" time="0.311" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testGetDealByID(DealsTestCases.java:54)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetDealByID -->
  <testcase name="testGetDealByID" time="0.341" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testGetDealByID(DealsTestCases.java:54)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetDealByID -->
  <testcase name="testBulkDelete" time="0.391" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testBulkDelete(DealsTestCases.java:99)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testBulkDelete -->
  <testcase name="testCreateDeal" time="2.401" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
  <testcase name="testCreateDealToContactUsingEmailID" time="2.719" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
  <testcase name="testGetDealsForParticularTrack" time="0.618" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testGetDealsForParticularTrack(DealsTestCases.java:126)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetDealsForParticularTrack -->
  <testcase name="testCreateContacts" time="3.047" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
  <testcase name="testBulkDelete" time="0.702" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testBulkDelete(DealsTestCases.java:99)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testBulkDelete -->
  <testcase name="testGetDealsRelateToSpecificContact" time="0.328" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
  <testcase name="testGetDealsForParticularTrack" time="0.527" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testGetDealsForParticularTrack(DealsTestCases.java:126)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetDealsForParticularTrack -->
  <testcase name="testListDeals" time="0.427" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;500&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<500>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testListDeals(DealsTestCases.java:45)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testListDeals -->
  <testcase name="testGetDealsForParticularTrack" time="0.539" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testGetDealsForParticularTrack(DealsTestCases.java:126)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetDealsForParticularTrack -->
  <testcase name="testGetDealsOfCurrentUser" time="1.297" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
  <testcase name="testListDeals" time="0.6" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;500&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<500>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testListDeals(DealsTestCases.java:45)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testListDeals -->
  <testcase name="testRemoveContactsOfDeal" time="0.47" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testRemoveContactsOfDeal(DealsTestCases.java:153)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testRemoveContactsOfDeal -->
  <testcase name="testUpdateDeal" time="0.27" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testUpdateDeal(DealsTestCases.java:72)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateDeal -->
  <testcase name="testRemoveContactsOfDeal" time="0.299" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testRemoveContactsOfDeal(DealsTestCases.java:153)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testRemoveContactsOfDeal -->
  <testcase name="testUpdateDeal" time="0.306" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testUpdateDeal(DealsTestCases.java:72)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateDeal -->
  <testcase name="testListDeals" time="0.575" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;500&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<500>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testListDeals(DealsTestCases.java:45)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testListDeals -->
  <testcase name="testRemoveContactsOfDeal" time="0.349" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testRemoveContactsOfDeal(DealsTestCases.java:153)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testRemoveContactsOfDeal -->
  <testcase name="testUpdateDeal" time="0.273" classname="com.zuora.interview.congjun.testcases.DealsTestCases">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;204&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<204>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at com.zuora.interview.congjun.testcases.DealsTestCases.testUpdateDeal(DealsTestCases.java:72)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateDeal -->
  <testcase name="testdeleteDeal" time="0.591" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
  <testcase name="testGetDealsForParticularTrackGroupByMilestone" time="24.975" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
  <testcase name="testGetDealsFromDefaultTracGroupByMilestones" time="29.506" classname="com.zuora.interview.congjun.testcases.DealsTestCases"/>
</testsuite> <!-- Deals Test -->
